site/content/community/guides.md

```diff
@@ -4,9 +4,11 @@ Share your applications, videos, and blog posts with fellow Copilots!
 
 | Title      | Description                          |
 | ----------- | ------------------------------------ |
-| [**Automatically deploying your container application with AWS Copilot**](https://aws.amazon.com/blogs/containers/automatically-deploying-your-container-application-with-aws-copilot/) by <a href="https://twitter.com/nathankpeck">@nathankpeck</a> | Nathan shows how to setup a release pipeline with the CLI that builds, pushes, and deploys an application. Finally, he sets up integration tests for validation before releasing to production. |
-| [**Deploying containers with the AWS Copilot CLI**](https://maartenbruntink.nl/blog/2020/08/16/deploying-containers-with-the-aws-copilot-cli-part-1/) by <a href="https://twitter.com/maartenbruntink">@maartenbruntink</a> | Maarten shows to deploy the [sample Docker voting app](https://github.com/dockersamples/example-voting-app) with the AWS Copilot CLI that showcases how to setup your own Redis and Postgres servers. In the [second part](https://maartenbruntink.nl/blog/2020/08/16/deploying-containers-with-the-aws-copilot-cli-part-2), he automates the release process. |
-| [**AWS Copilot: an application-first CLI for containers on AWS**](https://aws.amazon.com/blogs/containers/aws-copilot-an-application-first-cli-for-containers-on-aws/) by <a href="https://twitter.com/efekarakus">@efekarakus</a> | Efe walks through the design tenets of the CLI, why they were chosen, how they map to Copilot features, and the vision for how the CLI will evolve in the future.  |
+| [**Pilot your containers like a boss with AWS Copilot!**](https://blog.flolight.dev/pilot-your-containers-like-a-boss-with-aws-copilot) by <a href="https://twitter.com/FlolightC">@FlolightC</a> | Florian simplifies getting started with AWS Copilot by demoing deployment of the [ECS CLI sample app](https://github.com/aws-samples/amazon-ecs-cli-sample-app) from start to finish, including links to both required and optional resources.
```

Comment:
nit: can we rename the link to [sample app] so that it doesn't include "ecs cli" 😄